api/nodes/serializers.py

```diff
@@ -273,6 +278,30 @@ class NodeDetailSerializer(NodeSerializer):
     id = IDField(source='_id', required=True)
 
 
+class NodeForksSerializer(NodeSerializer):
+
+    category_choices = Node.CATEGORY_MAP.items()
+    category_choices_string = ', '.join(["'{}'".format(choice[0]) for choice in category_choices])
+
+    title = ser.CharField(required=False)
+    category = ser.ChoiceField(read_only=True, choices=category_choices, help_text="Choices: " + category_choices_string)
+    forked_date = ser.DateTimeField(read_only=True)
+
+    def create(self, validated_data):
+        node = validated_data.pop('node')
```

Comment:
Functional testing seems to indicate that fields that are not the title are not carrying over. 


```json
{
    "data": {
        "type": "nodes",
        "attributes": {
            "title": "Quicker test with category and description",
            "category": "project",
            "description": "A super-quick test"
        }
    }
}
```
Is creating a node with a new title but not a new category/description.